<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_commands">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Commands</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_execution_time">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Execution Time</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="edit_execution_time">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_execution_date">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Execution Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="edit_execution_date">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinbox_period">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_iteration_period">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>106</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Iteration Period</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hours">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hour(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_execution">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="mouse_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Mouse</string>
   </property>
   <widget class="QPushButton" name="button_left_click">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Click</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_right_click">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Click</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_drag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Drag</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="text_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Text Input</string>
   </property>
   <widget class="QPushButton" name="button_add_text">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="scroll_input">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>280</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Scroll</string>
   </property>
   <widget class="QPushButton" name="button_scroll_up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scroll Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_scroll_down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scroll Down</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinbox_scroll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="keyboard_input">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>280</y>
     <width>121</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Keyboard</string>
   </property>
   <widget class="QPushButton" name="button_press_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Press Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_release_key">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Release Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_release_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Release All</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_auto_click">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Auto Click</string>
   </property>
   <widget class="QDoubleSpinBox" name="edit_click_interval">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_second_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Second(s)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_calender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Calender Click</string>
   </property>
   <widget class="QGroupBox" name="group_register">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Register</string>
    </property>
    <widget class="QPushButton" name="button_1st_day">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1st day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_14th_day">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>14th day</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_later">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Later</string>
    </property>
    <widget class="QSpinBox" name="spinbox_month">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_month">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Month</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>25</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinbox_day">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>group_later</zorder>
   <zorder>group_register</zorder>
  </widget>
  <widget class="QGroupBox" name="input_interval">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>280</y>
     <width>83</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Interval</string>
   </property>
   <widget class="QDoubleSpinBox" name="spinbox_interval">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_second">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Second(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_command">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>80</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table_cmd_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Command</string>
    </property>
   </column>
  </widget>
  <zorder>scroll_input</zorder>
  <zorder>label_commands</zorder>
  <zorder>label_execution_time</zorder>
  <zorder>edit_execution_time</zorder>
  <zorder>label_execution_date</zorder>
  <zorder>edit_execution_date</zorder>
  <zorder>spinbox_period</zorder>
  <zorder>label_iteration_period</zorder>
  <zorder>label_hours</zorder>
  <zorder>button_execution</zorder>
  <zorder>mouse_input</zorder>
  <zorder>text_input</zorder>
  <zorder>keyboard_input</zorder>
  <zorder>group_auto_click</zorder>
  <zorder>group_calender</zorder>
  <zorder>input_interval</zorder>
  <zorder>table_cmd_list</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
